src/tools/icn2img.cpp

```diff
@@ -26,52 +26,55 @@
 #include <sstream>
 
 #include "agg_file.h"
+#include "image_palette.h"
 #include "image_tool.h"
 #include "serialize.h"
 #include "system.h"
 
 int main( int argc, char ** argv )
 {
-    if ( argc < 3 ) {
-        std::cout << argv[0] << " infile.icn extract_to_dir" << std::endl;
-        return EXIT_SUCCESS;
+    if ( argc < 4 ) {
```

Comment:
```suggestion
    if ( argc != 4 ) {
```

Extra parameters should not be accepted either.